packages/network-of-terms-reconciliation/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@netwerk-digitaal-erfgoed/network-of-terms-reconciliation",
+  "description": "A Reconciliation API for the Network of Terms",
+  "author": "Netwerk Digitaal Erfgoed",
+  "bugs": {
+    "url": "https://github.com/netwerk-digitaal-erfgoed/network-of-terms/issues"
+  },
+  "dependencies": {
+    "@hapi/hoek": "^9.2.1",
+    "@netwerk-digitaal-erfgoed/network-of-terms-catalog": "*",
+    "@netwerk-digitaal-erfgoed/network-of-terms-query": "*",
+    "fastify": "^3.27.1",
+    "fastify-accepts": "^2.1.0",
+    "fastify-cors": "^6.0.2",
+    "fastify-formbody": "^5.2.0",
+    "rdf-js": "^4.0.2"
+  },
+  "devDependencies": {
+    "@types/accepts": "^1.3.5",
+    "jest-dev-server": "^6.0.3"
+  },
+  "files": ["build/"],
+  "homepage": "https://github.com/netwerk-digitaal-erfgoed/network-of-terms#readme",
+  "license": "EUPL-1.2",
+  "main": "build/main.js",
+  "private": true,
```

Comment:
Unsure: does `private: true` still apply?

Reply:
It does: `network-of-terms-reconciliation` is an application to be deployed (as a Docker image) rather than a package (used in other Node projects).

Another way to look at this: if you’re building a Node app, it doesn’t make sense to include `network-of-terms-reconciliation`, which includes its own Fastify server etc. Instead, you would depend on `network-of-terms-query` (which _is_ published as an NPM package) and call that.

Reply:
👍